frontend/src/components/InstanceView.jsx

```diff
@@ -59,47 +59,51 @@ export const InstanceView = ({ templateName, objectID, endpoint }) => {
-        isHovering ? " shadow-md" : ""
-      }`}
-      onMouseEnter={() => setIsHovering(true)}
-      onMouseLeave={() => setIsHovering(false)}
-    >
-      {loading ? (
-        <div className="flex h-screen -translate-y-48 transform items-center justify-center">
-          <Spinner />
-        </div>
-      ) : (
-        <>
-          {isHovering && (
-            <div className="fixed -ml-14 -mt-6">
-              <div
-                onClick={() => window.print()}
-                className="flex cursor-pointer items-center justify-center rounded-full bg-custom-blue p-2 text-white transition-colors duration-700 ease-in-out hover:bg-custom-dark-4 dark:bg-custom-blue dark:text-gray-100 dark:hover:bg-custom-light"
-              >
-                <Printer></Printer>
-              </div>
+    <>
+      <div
+        ref={contentRef}
+        className={`html-content mb-4 rounded-lg border-4 border-transparent bg-gray-100 p-8 text-gray-700 shadow-lg  scrollbar scrollbar-track-gray-200 scrollbar-thumb-gray-400 hover:border-gray-300 dark:bg-custom-dark-2 dark:text-gray-100 dark:shadow-dark dark:scrollbar-track-custom-dark-3 dark:scrollbar-thumb-slate-600 md:w-[210mm] print:m-0 print:bg-transparent print:p-0 print:shadow-none ${
+          isHovering ? " shadow-md" : ""
+        }`}
+        onMouseEnter={() => setIsHovering(true)}
+        onMouseLeave={() => setIsHovering(false)}
+      >
+        {isHovering && (
+          <div className="fixed hidden md:-ml-14 md:-mt-6 md:block">
+            <div
+              onClick={() => window.print()}
+              className="flex cursor-pointer items-center justify-center rounded-full bg-custom-blue p-2 text-white transition-colors duration-700 ease-in-out hover:bg-custom-dark-4 dark:bg-custom-blue dark:text-gray-100 dark:hover:bg-custom-light"
+            >
+              <Printer></Printer>
             </div>
-          )}
-          {details.map((item, idx) => {
-            if (item.type === "SVGDisplay") {
-              return <SVGDisplay key={idx} content={item.content} />;
-            } else {
-              return (
-                <div
-                  key={idx}
-                  dangerouslySetInnerHTML={{
-                    __html: item.content,
-                  }}
-                />
-              );
-            }
-          })}
-        </>
-      )}
-    </div>
+          </div>
+        )}
+        {details.map((item, idx) => {
+          if (item.type === "SVGDisplay") {
+            return <SVGDisplay key={idx} content={item.content} />;
+          } else {
+            return (
+              <div
+                key={idx}
+                dangerouslySetInnerHTML={{
+                  __html: item.content,
+                }}
+              />
+            );
+          }
+        })}
+      </div>
+      <div className="mb-8 mt-8 text-center 3xl:hidden">
```

Comment:
This is related to the hacky/dumb approach i mentioned in the daily the other day.
Currently im displaying the version on the bottom left corner in the TemplateView when on bigger screens.
When an InstanceView is loaded the pages height changes dynamically which therefore makes it hard for me to place the version at the middle and bottom (underneath) the instance from within TemplateView. That why i also load in in InstanceView since i have easy access to the Height of the page. 

I'm Open for suggestions 😄